dojson/errors.py

```diff
@@ -26,6 +26,14 @@ class IgnoreKey(DoJSONException):
     """
 
 
+class IgnoreElem(DoJSONException):
+    """The corresponding elem for the current list key has been ignored.
+
+    .. versionadded:: ??
+
```

Comment:
Good. Then please remove it so we don't have unnecessary empty lines in the code that are not helping with readability.